content/docs/deployment/csminstallationwizard/_index.md

```diff
@@ -108,13 +106,11 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 >NOTE: The CSM Installation Wizard generates `values.yaml` with the minimal inputs required to install the CSM. To configure additional parameters in values.yaml, you can follow the steps outlined in [PowerStore](../../csidriver/installation/helm/powerstore/#install-the-driver), [PowerMax](../../csidriver/installation/helm/powermax/#install-the-driver), [PowerScale](../../csidriver/installation/helm/isilon/#install-the-driver), [PowerFlex](../../csidriver/installation/helm/powerflex/#install-the-driver), [Unity XT](../../csidriver/installation/helm/unity/#install-csi-driver), [Observability](../../observability/), [Replication](../../replication/), [Resiliency](../../resiliency/).
```

Comment:
@coulof Previous Note for Operator method had links to Operator doc section of drivers and modules (old line # 117). Newer note refers to the ones same as Helm method. Should we revert these links?

Reply:
Reverted the links.